### PR TITLE
CV2-3435 [full ci] adds support for presto

### PR DESCRIPTION
**Pull request by DGaffney**
## Description

Adds support for presto into check-web

References: 3435

## Type of change

- [ ] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Currently not tested - just adding modifications as per conversation with @caiosba 

## Things to pay attention to during code review

My first push into check-web so please let me know!

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I've made sure my branch is runnable and given good testing steps in the PR description
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] I have removed the /* eslint-disable @calm/react-intl/missing-attribute */ from any files I've worked on and added a `description` prop to all `<FormattedMessage />` components that were missing it.
- [ ] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

File: docker-test.yml [hunk 1]
```diff
@@ -38,6 +38,10 @@ services:
     environment:
       BOILERPLATE_ENV: test
       ALEGRE_PORT: 3100
+  presto:
+    environment:
+      BOILERPLATE_ENV: test
+      ALEGRE_PORT: 8000
```
Comment (caiosba) [inline docker-test.yml:44]: `ALEGRE_PORT`? 8000 is the Presto port, no?

Comment (DGaffney) [reply docker-test.yml]: whoops! Fixed

**Comment by DGaffney** [comment]
@dmou & @caiosba this is now failing because, I think, there's no `develop` branch in `presto` - but that is also (as I understand) a feature and not a bug of the new setup for this codebase (i.e. we are only building off master now). What should we do here? What can I do to resolve what seems to be a tension with the rest of the setup? Should we just `echo 'master'` instead of `echo 'develop'` in the `.travis.yml`?

**Comment by dmou** [comment]
> > @dmou & @caiosba this is now failing because, I think, there's no `develop` branch in `presto` - but that is also (as I understand) a feature and not a bug of the new setup for this codebase (i.e. we are only building off master now). What should we do here? What can I do to resolve what seems to be a tension with the rest of the setup? Should we just `echo 'master'` instead of `echo 'develop'` in the `.travis.yml`?
> 
> Eventually we're going to have a `develop` branch, no?

uh we had determined like a year ago that we were moving toward single branch deployments, which would be a master/main branch, and not a develop/qa branch, so that we can have actual trunk based development. are we walking that back?

**Comment by caiosba** [comment]
> > > @dmou & @caiosba this is now failing because, I think, there's no `develop` branch in `presto` - but that is also (as I understand) a feature and not a bug of the new setup for this codebase (i.e. we are only building off master now). What should we do here? What can I do to resolve what seems to be a tension with the rest of the setup? Should we just `echo 'master'` instead of `echo 'develop'` in the `.travis.yml`?
> > 
> > 
> > Eventually we're going to have a `develop` branch, no?
> 
> uh we had determined like a year ago that we were moving toward single branch deployments, which would be a master/main branch, and not a develop/qa branch, so that we can have actual trunk based development. are we walking that back?

Oh, yeah... I forgot about it. @DGaffney , since Check Web (and all other softwares of the suite) follow the "old" deployment way, I suggest we just fallback to `master` or `main`when there is no `develop`.

**Comment by DGaffney** [comment]
Sounds good @dmou and @caiosba - thank you for resolving!

**Review by caiosba — APPROVED** [review]
Good to go! Let's please merge this one last, @DGaffney (after Check API and Alegre).

**Pull request merged**